### PR TITLE
settings for vrgowiki

### PR DESCRIPTION
**Pull request by Reception123**
BLOCKED by miraheze/puppet/issues/COMING

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1198,6 +1199,14 @@
...
 			),
 		), 
+		'+vrgowiki' => array(
+			'sysop' => array(
+				'edit' => true,
+			'bureaucrat => array(
+				'edit' => true,
+			),	
+		),
+	),
```
Comment (Southparkfan) [inline LocalSettings.php]: ), for no reason?

File: LocalSettings.php [hunk 2]
```diff
@@ -1198,6 +1199,14 @@
 				'read' => true,
 			),
 		), 
+		'+vrgowiki' => array(
+			'sysop' => array(
+				'edit' => true,
+			'bureaucrat => array(
```
Comment (Southparkfan) [inline LocalSettings.php]: Missing a ),

**Comment by Reception123** [comment]
@Southparkfan what do you mean by "), for no reason"?

**Comment by NDKilla** [comment]
Done in #304

**Pull request closed without merging**